README.md

```diff
@@ -1,10 +1,11 @@
 # Terraform Provider
 
+![Gitter](https://img.shields.io/gitter/room/DataDog/hashicorp-terraform)
+
 -   Website: https://www.terraform.io
--   [![Gitter chat](https://badges.gitter.im/hashicorp-terraform/Lobby.png)](https://gitter.im/hashicorp-terraform/Lobby)
 -   Mailing list: [Google Groups](http://groups.google.com/group/terraform-tool)
 
-<img src="https://cdn.rawgit.com/hashicorp/terraform-website/master/content/source/assets/images/logo-hashicorp.svg" width="600px">
+<img alt="Terraform" src="https://www.datocms-assets.com/2885/1629941242-logo-terraform-main.svg" width="600px">
```

Comment:
This is the same link as the official `hashicorp/terraform` repository:

- https://github.com/hashicorp/terraform/blob/main/README.md?plain=1